### PR TITLE
Add docs page

### PR DESCRIPTION
**Pull request by Sladuca**
* started docs sub-repo (it has its own npm modules)
* wrote some stuff about `ChainInterface`
* wrote a "getting started" guide
* removed most of the docuasaurus stuff but not all of it because idc about having the docs fit the brand rn

You can run it by doing `npm run start` in the `docs` directory

### CODE REVIEW

File: docs/docs/intro.md [hunk 1]
```diff
@@ -2,34 +2,39 @@
...
 
-npx docusaurus start
-```
+Prerequisites: have `npm` and `near-cli` installed. 
 
-Your site starts at `http://localhost:3000`.
+1. `npm i` in the root of this repo
+2. If you're using the API, DM Lev or Sebastien to get the development client secret
+3. make a copy of `.env.example` in `apps/api/src/environments` and rename it `.env.dev`
+4. populate all of the fields in `.env.dev` with non-example values
```
Comment (Lev-Stambler) [inline docs/docs/intro.md:33]: This looks great but there is also a .env.dev in libs/baf-contract

File: docs/docs/libs/interfaces.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 1
+---
+
+# Purpose
```
Comment (Lev-Stambler) [inline docs/docs/libs/interfaces.md:5]: Having the section named purpose makes me think that its about like philosophy. (Like the button for next says Purpose). Could we change it to Interfaces or something?

Screenshot from 2021-05-03 20-01-53

Comment (Lev-Stambler) [reply docs/docs/libs/interfaces.md:5]: Or like Library

**Review by Lev-Stambler — APPROVED** [review]
This looks fantastic! There are just a few small things (mainly that .env.dev part). But approved pending changes

File: docs/docs/libs/interfaces.md (hunk 2 above)
Comment (Sladuca) [reply docs/docs/libs/interfaces.md:5]: Oh yes, I meant that to be Interfaces, I must have missed that

**Pull request merged**